Source/Scene/ClippingPlanesCollection.js

```diff
@@ -0,0 +1,189 @@
+define([
+    '../Core/BoundingSphere',
+    '../Core/buildModuleUrl',
+    '../Core/Cartesian3',
+    '../Core/Cartographic',
+    '../Core/Color',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/Intersect',
+    '../Core/Matrix4',
+    '../Core/Plane'
+], function(
+    BoundingSphere,
+    buildModuleUrl,
+    Cartesian3,
+    Cartographic,
+    Color,
+    defaultValue,
+    defined,
+    Intersect,
+    Matrix4,
+    Plane) {
+    'use strict';
+
+    /**
+     * Contains the options to specify a set of clipping planes. Clipping planes selectively disable rendering on an object on the outside of the specified list of {@link Plane}.
+     *
+     * @alias ClippingPlanesCollection
+     * @constructor
+     *
+     * @param {Object} [options] Object with the following properties:
+     * @param {Plane[]} [options.planes=[]] An array of up to 6 {@link Plane} used to selectively disable rendering on the outside of each plane.
+     * @param {Boolean} [options.enabled=true] Determines whether the clipping planes are active.
+     * @param {Matrix4} [options.transformationMatrix=Matrix4.IDENTITY] The 4x4 transformation matrix specifying an additional transform relative to the clipping planes original coordinate system.
+     * @param {Boolean} [options.inclusive=true] If true, the region to be clipped must be included in all planes in this collection. Otherwise, a region will be clipped if included in any plane in the collection.
```

Comment:
> inclusive

Is this the right name?  It is basically logical `OR` vs. `AND`.  A new enum would provide the most precision, but is overkill.  Is there a better name for a boolean?